core/domain/activity_jobs_one_off.py

```diff
@@ -628,3 +632,128 @@ def reduce(key, values):
             yield (key, values)
         else:
             yield (key, len(values))
+
+
+class AddMissingCommitLogsJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that adds the missing commit log entry model for the corresponding
```

Comment:
Can you add a link here to some sort of context that include & describe the findings? E.g. is there an issue or Google doc that you can point to?

Reply:
There is no particular issue or a google doc, so I have given more context in the comment.